### PR TITLE
chore: add release please for npm publishing

### LINKED ISSUE #29: Set up release please
## Description

The project needs to get the release please action configured. Secrets for release please are already available in the repository.

### PR DESCRIPTION
**Pull request by nickytonline**
## Description

Adds release please for automating `npm publish`.

As usual, ignore the `deploy/netlify — Deploy Preview failed.`. It's on my TODO to sort out.

## Related Tickets & Documents

- Related Issue #
- Closes #29

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes_

For us to review and ship your PR efficiently, please perform the following steps:

- [x] Open a [bug/issue](https://github.com/netlify/cli/issues/new/choose) before writing your code 🧑‍💻. This ensures we
      can discuss the changes and get feedback from everyone that should be involved. If you\`re fixing a typo or
      something that\`s on fire 🔥 (e.g. incident related), you can skip this step.
- [x] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 📖. This ensures your code follows our style
      guide and passes our tests.
- [ ] Update or add tests (if any source code was changed or added) 🧪
- [ ] Update or add documentation (if features were changed or added) 📝
- [x] Make sure the status checks below are successful ✅

**A picture of a cute animal (not mandatory, but encouraged)**

A hamster all bundled up eating a carrot cross section

### CODE REVIEW

File: .github/workflows/pre-release.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: prerelease
```
Comment (ascorbic) [inline .github/workflows/pre-release.yml:1]: I'm not sure this workflow will actually do anything. It's targeting the root directory, not the packages themselves. We've never updated the Next.js one to work with a monorepo, which is presumably where you got this? I'd remove it

Comment (nickytonline) [reply .github/workflows/pre-release.yml:1]: Will do.

File: release-please-config.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "bootstrap-sha": "62a4ce414f5b02f539902f88dd4b548df5cfa8ef",
```
Comment (nickytonline) [inline release-please-config.json:2]: Does this bootstrap SHA need to be diffent @ascorbic? I copied this config from the next-runtime project.

Comment (nickytonline) [reply release-please-config.json:2]: Looks like I need to generate this with the release please.

Comment (sarahetter) [reply release-please-config.json:2]: https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md#starting-commit

Looks like this isn't required, and is only added when you want to start the release please process from a particular git SHA. I'd use the one that main is at currently.

**Review by sarahetter — APPROVED** [review]

**Pull request merged**